client/src/pages/AgencyHomePage/AgencyHomePage.component.tsx

```diff
@@ -238,7 +212,7 @@ const AgencyHomePage = (): JSX.Element => {
         ? homePageDefaultView // no topics param key -> default homepage
         : !topicQueried
         ? homePageAllQuestionsView // topics param key + no topic queried -> all questions
-        : deviceType === device.desktop // specific topic selected
+        : deviceType === 'desktop' // specific topic selected
```

Comment:
we shouldn't be hardcoding string literals here, as we then lose the constraints that limit the comparison of `deviceType` to specific values. import the proposed enum mentioned elsewhere, and use those values instead.

Reply:
okay, I will make the suggested change, thank you for pointing that out.